common/fixation_entry.go

```diff
@@ -511,14 +512,14 @@ func (fs *FixationStore) getUnmarshaledEntryForBlock(ctx sdk.Context, safeIndex
 	return types.Entry{}, false
 }
 
-// FindEntry returns the entry by index and block without changing the refcount
-func (fs *FixationStore) FindEntry(ctx sdk.Context, index string, block uint64, entryData codec.ProtoMarshaler) bool {
+// FindEntry2 returns the entry by index and block without changing the refcount
+func (fs *FixationStore) FindEntry2(ctx sdk.Context, index string, block uint64, entryData codec.ProtoMarshaler) (uint64, bool) {
```

Comment:
we need a more concrete name for this

Reply:
It's the FindEntry with 2 return values :)